src/dbus_mpris.rs

```diff
@@ -296,8 +296,10 @@ async fn create_dbus_server(
                         // MPRIS spec: negative values should be treated as 0
                         let new_pos = new_pos.max(0);
                         if new_pos <= duration as i64 {
-                            let _ =
-                                sp_client.seek_track(new_pos as u32, playback.device.id.as_deref());
+                            let _ = sp_client.seek_track(
+                                Duration::milliseconds(new_pos),
```

Comment:
Haven't thought about it properly, but now that they're using Durations for `seek_track` (again?), would it maybe be nicer to convert our input (`new_pos`) to a `Duration` and do the addition with the `Duration`s?